tests/integration/awslambda/test_lambda.py

```diff
@@ -377,6 +379,31 @@ def test_runtime_introspection_arm(self, lambda_client, create_lambda_function,
         invoke_result = lambda_client.invoke(FunctionName=func_name)
         snapshot.match("invoke_runtime_arm_introspection", invoke_result)
 
+    @pytest.mark.skipif(
+        not use_docker(),
+        reason="Monkeypatching of Docker flags not applicable if run locally",
+    )
+    @pytest.mark.skip_snapshot_verify(condition=is_old_provider, paths=["$..LogResult"])
+    @pytest.mark.aws_validated
+    def test_runtime_ulimits(self, lambda_client, create_lambda_function, snapshot, monkeypatch):
+        """We consider ulimits parity as opt-in because development environments could hit these limits unlike in
+        optimized production deployments."""
+        monkeypatch.setattr(
+            config,
+            "LAMBDA_DOCKER_FLAGS",
+            "--ulimit nofile=1024:1024 --ulimit nproc=735:735 --ulimit core=-1:-1 --ulimit stack=8388608:-1",
```

Comment:
minimal list to pass the tests but we could also be more verbose here and cover the [snapshotted limits](https://github.com/localstack/localstack/pull/7871/files#diff-fadc6153815bbe901d871c5602204e98fbfe7f8fa264f686b0fa89f55c90765fR2667) to reduce potential environment-dependent flakes